examples/sliding-windows/src/main/java/com/hazelcast/jet/examples/slidingwindow/TopNStocks.java

```diff
@@ -116,9 +116,9 @@ public static void main(String[] args) throws Exception {
         public String toString() {
             return String.format(
                     "Top rising stocks:%n%s\nTop falling stocks:%n%s",
-                    topIncrease.stream().map(kwr -> String.format("   %s by %.2f", kwr.key(), kwr.result()))
+                    topIncrease.stream().map(kwr -> String.format("   %s by %.5f", kwr.key(), kwr.result()))
                                .collect(joining("\n")),
-                    topDecrease.stream().map(kwr -> String.format("   %s by %.2f", kwr.key(), kwr.result()))
+                    topDecrease.stream().map(kwr -> String.format("   %s by %.5f", kwr.key(), kwr.result()))
```

Comment:
Can't we make the price differences larger instead of adding decimals?

Reply:
Generally, we can. However it is currently hard-coded in `TradeSource` which is used on couple of different places and in demos etc. We can probably make it configurable but it needs to be documented etc. @jbartok you made refactoring related to `TradeSource` - what do you think about it? Can you take care of it?

Reply:
Yes, will do, on Monday.